lib/rom/sql/attribute.rb

```diff
@@ -235,6 +257,11 @@ def sql_expr
         @sql_expr ||= (meta[:sql_expr] || Sequel[to_sym])
       end
 
+      # @api private
+      def type_extensions
+        @type_extensions ||= TypeExtensions[unwrap_type(type)] || EMPTY_HASH
+      end
```

Comment:
oh, I wasn't thinking about run-time attribute creating, could you show me where this happens? Things like `int::count` I suppose...